src/NzbDrone.Core/Download/Clients/LibTorrent/Models/LibTorrentInfoHash.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Download.Clients.LibTorrent.Models
+{
+    // TODO: Figure out correct json serilization of `null` values, currently setting them to ""
+    public class LibTorrentInfoHash
+    {
+        #nullable enable
+        public string? Hash { get; set; }
```

Comment:
Strings are nullable by default.

Reply:
Yeah I need to rethink this class.
I don't have anything to test against so I don 't know how the distinction between a proper c++ `null` vs a C# _string null_ will behave.

I'll keep it C# for now and then someone can figure it out later when it's relevant